packages/webpack/src/index.js

```diff
@@ -26,11 +27,38 @@ class ReloadPlugin {
 			}
 		});
 
-		compiler.hooks.compilation.tap(this.constructor.name, compilation => {
-			compilation.mainTemplate.hooks.require.tap(
-				this.constructor.name,
-				(source, chunk, hash) =>
-					createRefreshTemplate(source, chunk, hash, compilation.mainTemplate)
+		// const providePlugin = new webpack.ProvidePlugin({
+		//   [prefreshUtils]: require.resolve('./utils/prefresh'),
+		// });
+
+		// providePlugin.apply(compiler);
```

Comment:
This was an attempt at abstracting out the logic used in the `loader-runtime` but alas I'm not succeeding